server/metrics/metrics.go

```diff
@@ -61,6 +62,9 @@ const (
 	// CRIOContainersOOMKey is the key for the CRI-O container out of memory metrics per container name.
 	CRIOContainersOOMKey = "crio_containers_oom"
 
+	// CRIODefunctProcessesTotalKey is the key for the total number of defunct processes in a node.
+	CRIODefunctProcessesTotalKey = "crio_defunct_processes_total"
```

Comment:
```suggestion
	CRIOProcessesDefunctTotalKey = "crio_processes_defunct_total"
```

Reply:
The `_total` suffix should only be used for counters, summaries and histograms (see https://prometheus.io/docs/instrumenting/writing_exporters/#naming).

```suggestion
	CRIODefunctProcessesTotalKey = "crio_defunct_processes"
```

Reply:
Good catch!